src/components/10-atoms/checkbox/index.react.d.ts

```diff
@@ -5,6 +5,7 @@ export interface AXACheckboxProps {
   value?: string;
   name?: string;
   label?: string;
+  variant?: string;
```

Comment:
If you have a set of allowed strings, write it that way: ` 'tick'|'square' `

Reply:
Thanks